scalafix-core/src/main/scala/scalafix/config/CustomMessage.scala

```diff
@@ -13,7 +13,9 @@ class CustomMessage[T](
 
 object CustomMessage {
   implicit val SymbolDecoder: ConfDecoder[CustomMessage[Symbol.Global]] =
-    decoder[Symbol.Global](field = "symbol")
+    decoder[Symbol.Global](field = "symbol")(
+      ScalafixMetaconfigReaders.symbolGlobalReader
+    )
```

Comment:
is that a no-op?

Reply:
yes

Reply:
the goal was just to know which implicit is used from ScalafixMetaconfigReaders